docs/migration/11.x-carbon-icons-react.md

```diff
@@ -0,0 +1,106 @@
+# `@carbon/icons-react`
+
+**Note: everything in this file is a work-in-progress and will be changed.**
+
+## Overview
+
+The `@carbon/icons-react` package has been updated to minimize the number of
+exports from the package to help reduce build and compile times.
+
+This update includes a change to the API of the icon components that come from
+this package. Previously, we would export icons that included the size of the
+asset. This update allows you to bring the icon directly and specify the size
+using the `size` prop.
+
+**Before**
+
+```jsx
+import { Add32, Add24, Add20, Add16 } from '@carbon/icons-react';
+
+function MyComponent() {
+  return (
+    <>
+      <Add32 />
+      <Add24 />
+      <Add20 />
+      <Add16 />
+    </>
+  );
+}
+```
+
+**After**
+
+```jsx
+import { Add } from '@carbon/icons-react';
+
+function MyComponent() {
+  return (
+    <>
+      <Add size={32} />
+      <Add size={24} />
+      <Add size={20} />
+      <Add />
+    </>
+  );
+}
+```
+
+_Note: by default, the `size` prop will point to the 32x32px version of the SVG
+asset_
```

Comment:
I think by default it's pointing to 16x16 versions. When I ran the script, only icons size 32 and 20 were receiving "size" props, where as 16 looked like default.
<img width="1305" alt="Screen Shot 2021-05-03 at 9 09 23 PM" src="https://user-images.githubusercontent.com/32556167/116954006-5ea47a80-ac54-11eb-8ea8-3a2072756369.png">
<img width="1292" alt="Screen Shot 2021-05-03 at 9 09 45 PM" src="https://user-images.githubusercontent.com/32556167/116954012-6106d480-ac54-11eb-8476-d5f1815eafa4.png">
<img width="1283" alt="Screen Shot 2021-05-03 at 9 10 07 PM" src="https://user-images.githubusercontent.com/32556167/116954016-62380180-ac54-11eb-909e-86d4412347ce.png">
<img width="1304" alt="Screen Shot 2021-05-03 at 9 10 43 PM" src="https://user-images.githubusercontent.com/32556167/116954018-62380180-ac54-11eb-94e0-8af4cf385903.png">
<img width="1270" alt="Screen Shot 2021-05-03 at 9 11 20 PM" src="https://user-images.githubusercontent.com/32556167/116954019-62380180-ac54-11eb-90fd-ba0d547119a1.png">

Reply:
Yes! Great catch, I should update 👍